modules/dnn/test/test_halide_nets.cpp

```diff
@@ -0,0 +1,120 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2017, Intel Corporation, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+namespace cvtest
+{
+
+using namespace cv;
+using namespace dnn;
+
+void loadNet(const std::string& weights, const std::string& proto,
```

Comment:
It's working somehow. Should I add it for prevent fails in different build cases?